src/tiled/tiledproxystyle.cpp

```diff
@@ -370,7 +370,7 @@ void TiledProxyStyle::drawPrimitive(PrimitiveElement element,
             int state = option->state;
 
             QColor pressedColor = mergedColors(option->palette.base().color(),
-                                               option->palette.foreground().color(), 85);
+                                               option->palette.windowText().color(), 85);
```

Comment:
Strange, according to the documentation, `foreground()` was apparently [already obsolete in Qt 5.6](https://doc.qt.io/archives/qt-5.6/qpalette-obsolete.html#foreground) or even earlier, but I'm compiling with `QT_DISABLE_DEPRECATED_BEFORE=0x050900` and this still worked... apparently it's not covered.